ghost/members-api/lib/controllers/RouterController.js

```diff
@@ -492,6 +492,13 @@ module.exports = class RouterController {
             });
         }
 
+        if (honeypot) {
```

Comment:
Can we log out in this block? Otherwise it's gonna be difficult to know when we're receiving honeypot'd requests

Reply:
Agreed, since the response is the same.

Reply:
Good shout, I'll add logging.